src/tabs.scss

```diff
@@ -12,16 +12,56 @@
 $block: #{$fd-namespace}-tabs;
 
 .#{$block} {
+  @mixin tabs-semantic($color) {
+    .#{$block}__icon {
```

Comment:
height should be 2.75rem

Reply:
And icon height should be 1.25rem

Reply:
It's the same as above. Height is 2.625rem + 0.125rem(from border)